src/test/acceptance/features/cookies.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Cookies details page
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  I want to view the cookies used by the site
+
+  Scenario: The cookie back link is not shown if we navigate directly to it
+    When I go directly to the cookies page
+    Then the cookies page is shown
+    And no back link is shown
+
+  Scenario: The cookie back link is shown and links to the last page in the application if we navigate to it via a link in the application
```

Comment:
How about: `The back link on the Cookie page is shown when the page is accessed via an application page`

Reply:
Done